src/main/java/com/eternalcode/parcellockers/parcel/ParcelRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,102 @@
+package com.eternalcode.parcellockers.parcel;
+
+import com.eternalcode.parcellockers.database.JdbcConnectionProvider;
+import lombok.SneakyThrows;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public class ParcelRepositoryJdbcImpl implements ParcelRepository {
+
+    private final JdbcConnectionProvider jdbcConnectionProvider;
+
+    private ParcelRepositoryJdbcImpl(JdbcConnectionProvider jdbcConnectionProvider) {
+        this.jdbcConnectionProvider = jdbcConnectionProvider;
+    }
+
+    @Override
+    public CompletableFuture<Void> save(Parcel parcel) {
+        return CompletableFuture.runAsync(() -> {
+            this.jdbcConnectionProvider.executeUpdate("INSERT INTO `parcels` (`uuid`, `name`, `description`, `priority`, `receiver`, `size`, `entryLocker`, `destinationLocker`, `sender`) VALUES ("
+                    + parcel.getUuid()
+                    + "," + parcel.getMeta().getName()
+                    + "," + parcel.getMeta().getDescription()
+                    + "," + parcel.getMeta().isPriority()
+                    + "," + parcel.getMeta().getReceiver()
+                    + "," + parcel.getMeta().getSize().name()
+                    + "," + parcel.getMeta().getEntryLocker().getUuid()
+                    + "," + parcel.getMeta().getDestinationLocker().getUuid()
+                    + "," + parcel.getSender() + " )");
+        });
+    }
```

Comment:
To co Ci pisałem wyżej robisz gotowego stringa może nawet static final i później metoda `String.format` tak by ja to zrobił